kratos/includes/initial_state.h

```diff
@@ -0,0 +1,279 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Alejandro Cornejo
+//
+//
+
+#if !defined(KRATOS_INITIAL_STATE_H_INCLUDED)
+#define  KRATOS_INITIAL_STATE_H_INCLUDED
+
+// System includes
+#include <iostream>
+#include <string>
```

Comment:
```suggestion
#include <iostream>
#include <string>
#include <atomic>

```

necessary for the intrusive ptr to work